fluentnamer/src/main/java/com/azure/autorest/fluent/util/FluentConsts.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.autorest.fluent.util;
+
+public class FluentConsts {
+
+    public static final String DEFAULT_NAME_FOR_UNGROUPED_OPERATIONS = "ResourceProvider";
+    public static final String ANTI_CONFLICT_SUFFIX_FOR_UNGROUPED_OPERATIONS = "Operation";
```

Comment:
Just `OPERATION_GROUP_DEDUPLICATE_SUFFIX`

We use the same "Operation" suffix in NamingConflictResolver.java for same purpose (if convenient, let that class use this constant as well).

Reply:
Changed.